### PR TITLE
[stable/fluent-bit] Upgrade to Fluent Bit v0.14.6

### PR DESCRIPTION
**Pull request by edsiper**
This patch upgrade the chart image to Fluent Bit v0.14.6. This
upgrade is strongly required since it fixes two memory leaks
found by users who does filtering.

Signed-off-by: Eduardo Silva <eduardo@treasure-data.com>

#### What this PR does / why we need it:

This patch upgrade the chart image to Fluent Bit v0.14.6. This upgrade is strongly required since it fixes two memory leaks found by users who does filtering.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/fluent-bit/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on_minikube: false
 image:
   fluent_bit:
     repository: fluent/fluent-bit
-    tag: 0.14.4
+    tag: 0.14.6
```
Comment (davidkarlsen) [inline stable/fluent-bit/values.yaml:8]: can you update the readme too please?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/fluent-bit/values.yaml (hunk 1 above)
Comment (edsiper) [reply stable/fluent-bit/values.yaml:8]: thanks, just updated.

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**